src/coreclr/inc/clrconfigvalues.h

```diff
@@ -732,6 +732,7 @@ RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableSSE41,                  W("EnableSSE41")
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableSSE42,                  W("EnableSSE42"),               1, "Allows SSE4.2+ hardware intrinsics to be disabled")
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableSSSE3,                  W("EnableSSSE3"),               1, "Allows SSSE3+ hardware intrinsics to be disabled")
 RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableX86Serialize,           W("EnableX86Serialize"),        1, "Allows X86Serialize+ hardware intrinsics to be disabled")
+RETAIL_CONFIG_DWORD_INFO(EXTERNAL_EnableAPX,                    W("EnableAPX"),                 1, "Allows APX+ features to be disabled")
```

Comment:
This config is unused in the current version of the PR and it can be deleted. Is that correct? 

(Only the one in the JIT used by altjits is used.)

Reply:
This config by itself doesn't turn APX on in 'normal workflow'. It has 2 purposes

Use 1 : It's a knob to turn APX on when using altjit unless otherwise specified([link](https://github.com/dotnet/runtime/pull/instructionSetFlags.AddInstructionSet(InstructionSet_EVEX);)). Out current design is to turn all available features ON with altjit

Use 2 : A knot to turn APX OFF even on machines supporting APX([link](https://github.com/dotnet/runtime/pull/108799/files#diff-27bc831b0fc655d38b44b1420f0827820f2bf349263ba71543ff3bcd05d4c354R10003-R10011))

Reply:
Ok, I see what you are trying to do. The Use 2 is not implemented correctly - see my other comment https://github.com/dotnet/runtime/pull/108799#discussion_r1929372951